src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.CountryDao;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
 
 public class CountryServiceImpl implements CountryService {
+    private final CountryDao countryDao;
+
+    public CountryServiceImpl(CountryDao countryDao) {
+        this.countryDao = countryDao;
+    }
+
     @Override
     public Country add(Country country) {
-        return null;
+        return countryDao.add(country);
     }
 
     @Override
     public Country get(Long id) {
-        return null;
+        return countryDao.get(id).orElseThrow(() ->
+                new DataProcessingException("Can't find country by id: " + id));
```

Comment:
Exception on dao layer means that something went wrong with DB communication and we use DataProcessingException to represent these problems.
On service layer situation is different. We have successful call to DB and just don't have requested data in DB. So we should use proper Exception that represents another case
```suggestion
                new EntityNotFoundException("Can't find country by id: " + id));
```
Please recheck all service methods for same issue